nulib/meadow
# Fix bug in uploading Ingest Sheet.  Bump React Media Player version

# Summary 
Fixes a bug when uploading an ingest sheet.  For some reason when handling the Dropzone component's file selection, the update state callback function was not firing, so on the actual Create Ingest Sheet mutation success callback, there was a erroneous reference a property on the `currentFile` state object.  Nothing in the code surrounding this has changed in a year.  Weird.

# Specific Changes in this PR
- Handle a missing object property reference for uploaded file title
- Update React Media Player version

# Version bump required by the PR

See [Semantic Versioning 2.0.0](https://semver.org/) for help discerning which is required.


# Steps to Test
- In the `assets` folder, do an `npm install`.
- Go to a Project in Meadow, and upload an Ingest Sheet.  It should upload without errors.


# :rocket: Deployment Notes

**Note** - __if you check any of these boxes go to the [(always open) `main` <- `staging` PR](https://github.com/nulib/meadow/pulls) and add detailed notes and instructions to help out others who may end up deploying your changes to production__

- Backward compatible API changes
- Backwards-incompatible API changes


# Tested/Verified
